apps/remix-ide/src/app/tabs/runTab/model/dropdownlogic.js

```diff
@@ -50,9 +51,11 @@ class DropdownLogic {
         } catch (e) {
           return cb('Failed to parse the current file as JSON ABI.')
         }
+        _paq.push(['trackEvent', 'udapp', 'AtAddressLoadWithABI'])
```

Comment:
what is the difference between that one and  `AtAddressFromABI` ?

Reply:
if compiled contract or .abi is used

Reply:
do you really need `_paq.push(['trackEvent', 'udapp', 'AtAddressFromABI'])` above?

Reply:
just to collect some more statistics. if you think it's not important I can remove it